acceptance-test/src/acceptance-test/java/org/zalando/nakadi/repository/kafka/KafkaTestHelper.java

```diff
@@ -118,8 +140,16 @@ public static String getTopicCleanupPolicy(final String topic, final String zkPa
     }
 
     public static String getTopicProperty(final String topic, final String zkPath, final String propertyName) {
-        final ZkUtils zkUtils = ZkUtils.apply(zkPath, 30000, 10000, false);
-        final Properties topicConfig = AdminUtils.fetchEntityConfig(zkUtils, ConfigType.Topic(), topic);
-        return topicConfig.getProperty(propertyName);
+        KafkaZkClient zkClient = null;
+        try {
+            zkClient = getZkClient(zkPath);
+            final AdminZkClient adminZkClient = new AdminZkClient(zkClient);
+            final Properties topicConfig = adminZkClient.fetchEntityConfig(ConfigType.Topic(), topic);
+            return topicConfig.getProperty(propertyName);
+        } finally {
+            if (zkClient != null) {
+                zkClient.close();
+            }
```

Comment:
The same as with previous - make it final, don't check for null (reduce amount of lines)